grpc-sys/build.rs

```diff
@@ -317,5 +334,43 @@ fn main() {
     cc.warnings_into_errors(true);
 
     cc.compile("libgrpc_wrap.a");
-    bindgen_grpc(bind_config);
+
+    // Determine if need to update bindings. Supported platforms
+    // do not need to be updated by default unless the
+    // UPDATE_BIND is specified. Other platforms use bindgen
+    // to generate the bindings every time.
+    match env::var("TARGET").unwrap_or("".to_owned()).as_str() {
+        "x86_64-unknown-linux-gnu" => {
+            let file_path = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap())
+                .join("bindings")
+                .join("x86_64-unknown-linux-gnu-bindings.rs");
+            if env::var("UPDATE_BIND").is_ok() {
+                bindgen_grpc(bind_config, &file_path);
+            }
+            println!(
+                "cargo:rustc-env=BINDING_DIR={}",
+                file_path.to_str().unwrap()
+            );
+        }
+        "x86_64-apple-darwin" => {
+            let file_path = PathBuf::from(env::var("CARGO_MANIFEST_DIR").unwrap())
+                .join("bindings")
+                .join("x86_64-apple-darwin-bindings.rs");
+            if env::var("UPDATE_BIND").is_ok() {
+                bindgen_grpc(bind_config, &file_path);
+            }
+            println!(
```

Comment:
Maybe you can put the print out of the match.

Reply:
you are right